src/storage/mod.rs

```diff
@@ -1163,41 +1163,56 @@ mod tests {
         assert_eq!(x.unwrap().unwrap(), v);
     }
 
+    fn expect_scan(v: Vec<Option<KvPair>>, x: Result<Vec<Result<KvPair>>>) {
+        let x: Vec<Option<KvPair>> = x.unwrap().into_iter().map(Result::ok).collect();
+        assert_eq!(x, v);
+    }
+
+    fn expect_batch_get_vals(v: Vec<Option<KvPair>>, x: Result<Vec<Result<KvPair>>>) {
```

Comment:
What's the difference with `expect_scan`?

Reply:
Oh yes. They are duplicates.